AsyncDisplayKit/ASTableView.mm

```diff
@@ -848,10 +894,15 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertNodes:(NSA
   if (!self.asyncDataSource) {
     return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
   }
-
+  
   BOOL preventAnimation = animationOptions == UITableViewRowAnimationNone;
-  ASPerformBlockWithoutAnimation(preventAnimation, ^{
+  ASPerformBlockWithoutAnimationCompletion(preventAnimation, ^{
     [super insertRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOptions];
+  }, ^{
```

Comment:
Instead of this occurring after the animation, does it work to dispatch it after the insert call within the blocked provided to the no animation API? Even in the case that an animation occurs, I believe all of the values are supposed to be updated to their final value. This would be important to check, but I think even the content side should reflect the new value. If true, it would also provide a performance advantage to do this, as slow network loads can be initiated about 300 ms sooner. But I think the biggest advantage would be lower complexity in understanding how this code works and not immediately requiring a adoption of a new type of no-animation handler

This is not a blocker for landing the change, so prioritize other updates to the change first and we can come back to this after fully testing the environment and accessibility APIs.